calico-enterprise/operations/monitor/metrics/recommended-metrics.mdx

```diff
@@ -0,0 +1,514 @@
+| Threshold breach symptoms        | N/A.                                                         |
+| Threshold breach recommendations | N/A.                                                         |
+| Priority level                   | Optional.                                                    |
+
+### Active calico nodes
+
+| Active calico nodes              |                                                              |
+| -------------------------------- | ------------------------------------------------------------ |
+| Metric                           | *max(felix_cluster_num_hosts)*                               |
+| Explanation                      | Total number of nodes in the cluster that have calico-node deployed and running. |
+| Threshold value recommendation   | This value should be equal to the number of nodes in the cluster. If there are discrepancies, then calico-nodes on some nodes are having issues. |
+| Threshold breach symptoms        | {{prodname}} network policies on affected nodes could be out-of-sync. |
+| Threshold breach recommendations | Check calico-node logs, rollout restart calico-node if needed. |
+| Priority level                   | Recommended.                                                 |
+
+### Felix cluster policies
+
+| Felix cluster policies           |                                                              |
+| -------------------------------- | ------------------------------------------------------------ |
+| Metric                           | *felix_cluster_num_policies*                                 |
+| Explanation                      | Total number of {{prodname}} network policies in the cluster. |
+| Threshold value recommendation   | Because {{prodname}} is a distributed system, the number of policies should be generally consistent across all nodes.  It is expected to have some skew between nodes for a short time while they sync, however, they should never be out of sync for very long. |
+| Threshold breach symptoms        | If nodes are out of sync for a long time, calico-nodes may be having issues or experiencing resource contention.  Check the Errors Plot to see if there are any iptables errors reported. |
+| Threshold breach recommendations | Redeploy calico-node if issues are seen, and increase resources if needed. |
+| Priority level                   | Optional.                                                    |
+
+
+### Felix active local policies
+
+| Felix active local policies      |                                                              |
+| -------------------------------- | ------------------------------------------------------------ |
+| Metric                           | *felix_active_local_policies*                                |
+| Explanation                      | Total number of network policies deployed on a per-node basis. |
+| Threshold value recommendation   | ??                                                           |
```

Comment:
There's no hard limit we can handle 1000+ active policies _but_ having 1000 active policies will impact performance, especially if there's pod churn.  Best to optimise policies by combining multiple rules into one policy and to make sure  that top-level policy selectors are being used.

Reply:
Added